services/streammanager/streammanager.go

```diff
@@ -21,6 +22,9 @@ func NewProducer(destinationConfig interface{}, destination string) (interface{}
 	case "AZURE_EVENT_HUB":
 		producer, err := kafka.NewProducerForAzureEventHub(destinationConfig)
 		return producer, err
+	case "FIREHOSE":
+		producer, err := firehose.NewProducer(destinationConfig)
+		return producer, err
 	default:
```

Comment:
As we are adding more and more these things, might be a better idea to keep these sorted in alphabetical order? Will be easier to get to the section when needed.